manifests/init.pp

```diff
@@ -289,6 +294,7 @@
     class { '::python':
       virtualenv => 'present',
       dev        => 'present',
+      use_epel   => $python_use_epel,
```

Comment:
This might also be backwards incompatible for any users that have already worked around the issue by setting `python::use_epel: false` in their hiera?

Reply:
I forgot how hiera handles this. If it's undef, won't it still use the hiera value?

Reply:
Why is it an optional boolean vs. just a boolean again?  And even if they had set `python::use_epel: false` earlier, would this actually generate an error if this setting were also false, or just be redundant?

Reply:
Ah I see, to try to avoid changing existing behavior. Makes sense.

Reply:
@ekohl Not sure.  It's probably not too much of a big deal.